circuit_knitting/cutting/qpd/instructions/qpd_gate.py

```diff
@@ -201,6 +201,11 @@ def _define(self) -> None:
             qc.append(CircuitInstruction(op, [qc.qubits[0]], []))
         self.definition = qc
 
+    @property
```

Comment:
Could you please just quickly explain this mechanism here? Not familiar with _directive

Reply:
It is something that @mtreinish told me about.  It tells the transpiler to treat it as something special that ought to be passed through most passes without being touched, and is used for `Barrier`s and a few other things.

See [this brief description](https://github.com/Qiskit/qiskit-terra/blob/e9f8b7c50968501e019d0cb426676ac606eb5a10/qiskit/circuit/instruction.py#L55-L57), which itself links to https://github.com/Qiskit/qiskit-terra/issues/5811